ereolen_support/js/clearFields.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * @file clearFields.js
+ *   Clear fields on form change.
+ */
+(function () {
+  Drupal.behaviors.ereolenClearFields = {
+    attach: function (context, settings) {
+      'use strict';
+
+      let problem = document.getElementById("edit-ereolen-support-problem");
+      clearFields();
+
+      problem.addEventListener('change', function(){
+        clearFields();
+      });
+    }
+  };
+
+  /**
+   * Clear fields.
+   */
+  function clearFields() {
+    let textFields = document.querySelectorAll('.form-type-textfield input');
```

Comment:
Minor: would link the let statement to be with the loop that use it an have an empty line before the comments to make it more readable.

Reply:
Changed the order